x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotsIntegTests.java

```diff
@@ -681,21 +681,29 @@ public void testSnapshotMountedIndexLeavesBlobsUntouched() throws Exception {
 
```

Comment:
The original timestamp PR had a couple assertions more added in this integtest, let us add those for event ingested timestamp too:

https://github.com/elastic/elasticsearch/pull/65689/files#diff-f1bb44f1e1546df7098770be6ad7127aea7fa2ff3e7d4601e90e4d3ad520944fR238

Reply:
Likewise for `FrozenIndexIT` and `FrozenIndexTests`.

Reply:
Thanks for flagging. Added in next push.